docs/eventing/features/authorization.md

```diff
@@ -0,0 +1,471 @@
+      - exact:
+          type: com.github.push
+  ```
+
+If the filters are specified, an event must match **all** the specified filters of an `EventPolicy` (in addition to its `.spec.from`) to be accepted. `.spec.filters` accepts the same filter dialects as [Triggers](../triggers/README.md#supported-filter-dialects).
+
+!!! note
+    Filters apply in addition to `.spec.from`. This means, soon as an `EventPolicy` specifies `.spec.filters`, they must match the request, as well as the `.spec.from` (*AND* operand). Only then the `EventPolicy` allows the request.
+
+### Summary of `.spec` fields:
+
+* `to.ref` targets a specific resource.
+* `to.selector` targets a set of resources based on labels.
+* `from.ref` authorizes a specific event source resource.
+* `from.sub` authorizes specific users, service accounts, or patterns of accounts.
+* `.spec.filters` allows to define advanced CloudEvent filtering criterias
+
+## EventPolicy status
+
+The status of an `EventPolicy` provides information about resolved sources and readiness:
+
+```yaml
+status:
+  from:
+    - system:serviceaccount:default:my-source-oidc-sources.knative.dev-pingsource
+    - system:serviceaccount:default:trusted-app
+    - "system:serviceaccount:default:other-*"
+  conditions:
+    - type: Ready
+      status: "True"
+    - type: SubjectsResolved
+      status: "True"
+```
+
+## Applying EventPolicies to Resources
+
+Event consumers, such as a `Broker`, will list the applying EventPolicies in their status:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: my-broker
+spec:
+  ...
+status:
+  ...
+  policies:
+    - name: my-event-policy
+      apiVersion: v1alpha1
+  conditions:
+    - type: Ready
+      status: "True"
+    - type: EventPoliciesReady
+      status: "True"
+```
+
+The `EventPoliciesReady` condition indicates whether all applicable EventPolicies for a resource are ready and have been successfully applied.
+
+## Rejection Behavior
+
+If a request does not pass any applicable `EventPolicy`, it will be rejected with a `403 Forbidden` HTTP status code, ensuring that unauthorized event deliveries are blocked.
+
+## Example
+
+In the following, we give a full example how to configure authorization for resources. In this example, we want to protect a Broker (`broker`) in `namespace-1` by only allowing requests from a PingSource (`pingsource-2`) which is in a different namespace (`namespace-2`).
+
+![Example Overview](./images/authz-example.png)
+
+First we create the Namespaces, Broker and PingSources:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: namespace-1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: namespace-2
+---
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: broker
+  namespace: namespace-1
+---
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: pingsource-1
+  namespace: namespace-1
```

Comment:
`namespace-2` ?

Reply:
:+1:

Reply:
Depending on the step in the example, events from this source will be allowed or rejected. So I tried to make it clearer in the different steps. I also tried to make it clear in the image, with the "architecture overview" (not sure, if you checked the code or the rendered output of this PR :smile:  )